build.ts

```diff
@@ -1,13 +1,36 @@
 import { build } from 'esbuild'
+import { readdirSync, writeFileSync, unlinkSync } from 'fs'
+import { join, extname, basename } from 'path'
 
-// Load environment variables from process.env
 const configStoreName = process.env.CONFIG_STORE_NAME || 'Fingerprint'
 
+function getPluginEntries(folder: string) {
+  const dirPath = join(__dirname, folder)
+  return readdirSync(dirPath)
+    .filter((file) => extname(file) === '.ts' && !basename(file).endsWith('.example.ts'))
+    .map((file) => `import '${join(dirPath, file).replace(/\\/g, '/')}';`)
+}
+
+const virtualEntry = './virtual-entry.ts'
+const pluginImports = getPluginEntries('./plugins').join('\n')
+const entryContent = `import './src/index.ts';\n${pluginImports}`
```

Comment:
I don't like the fact that we are creating a file during build time, writing code in it and including it in the build time. This is error prone, and too complicated. I've created a counter-offer in the form of a PR [here](https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/pull/11)